### PR TITLE
Salt SSH test fixes on RHELL

### PR DESCRIPTION
**Pull request by isbm**

### CODE REVIEW

File: tests/ssh/test_ssh.py [hunk 1]
```diff
@@ -68,25 +68,47 @@ def test_ssh_cmdrun(master):
...
 
 def test_ssh_pkg_install(master):
     '''
     Test pkg.install
     '''
-    master.salt_ssh("cmd.run 'zypper --non-interactive rm test-package'")
+    master.salt_ssh("cmd.run 'rpm -e test-package'")
     out = master.salt_ssh("pkg.install test-package")
     assert bool(out.get('test-package', {}).get('new'))
     assert not bool(out.get('test-package', {}).get('old'))
```
Comment (dincamihai) [inline tests/ssh/test_ssh.py]: @isbm please remove the `bool()` wrapping if not needed

File: tests/ssh/test_ssh.py [hunk 2]
```diff
@@ -68,25 +68,47 @@ def test_ssh_cmdrun(master):
...
 
 
-def test_ssh_pkg_remove(master):
+@pytest.mark.tags('rhel', 'fedora')
+def test_ssh_pkg_remove_rhel(master):
+    '''
+    Test pkg.remove on RHEL
+    '''
+    master.salt_ssh("cmd.run 'yum install test-package -y'")
```
Comment (dincamihai) [inline tests/ssh/test_ssh.py]: @isbm why not using `master.salt_ssh("pkg.install...` here and have a single test that works both in sles and rhel?

File: tests/ssh/test_ssh.py [hunk 3]
```diff
@@ -68,25 +68,47 @@ def test_ssh_cmdrun(master):
...
     '''
-    master.salt_ssh("cmd.run 'zypper --non-interactive rm test-package'")
+    master.salt_ssh("cmd.run 'rpm -e test-package'")
     out = master.salt_ssh("pkg.install test-package")
     assert bool(out.get('test-package', {}).get('new'))
     assert not bool(out.get('test-package', {}).get('old'))
 
 
-def test_ssh_pkg_remove(master):
+@pytest.mark.tags('rhel', 'fedora')
```
Comment (dincamihai) [inline tests/ssh/test_ssh.py]: @isbm there's no 'fedora' tag anywhere and we don't run the tests with fedora. please remove it from here.

File: tests/ssh/test_ssh.py (hunk 2 above)
Comment (isbm) [inline tests/ssh/test_ssh.py]: Because if `pkg.install` it is broken. no more test for `pkg.remove`.

**Comment by dincamihai** [comment]
Thanks for fixing this.

**Pull request merged**